README.md

```diff
@@ -62,95 +61,54 @@ Command Palette (^ means the `ctrl` key):
 |Go to definition  | `^T^D` or `F12` |
 |Paste and format  | `^V` or <code>&#8984;V</code> |
 |Quick info        | `^T` `^Q`       |
+|Build		   | (Win)`^B` or `F7`, (OSX) `⌘B` or `F7`   |
+|Error list        | (via Command Palette) |
+
+Project System
+------
+The plugin supports two kinds of projects:
 
-The plugin supports representing a TypeScript project via a 
-[tsconfig.json](https://github.com/Microsoft/TypeScript/pull/1692) file. If a 
-file of this name is detected in a parent directory, then its settings will be 
+- Inferred project
+For loose TS files opened in Sublime, the plugin will create an inferred project and include every files that the current file refers to;
+
+- Configured project
+The plugin also supports representing a TypeScript project via a [tsconfig.json](https://github.com/Microsoft/TypeScript/pull/1692) file. If a file of this name is detected in a parent directory, then its settings will be 
```

Comment:
Line 72 will be on the same line with line 71.
Same thing happens for line 75 and 74.